src/pages/index.tsx

```diff
@@ -17,6 +20,8 @@ const Home: WithPageLayout<HomeProps> = () => {
           content="TechIsHiring - A space where great jobs and great devs can find each other!"
         />
       </Head>
+      {/* <Hero /> */}
+      <HeroStory />
```

Comment:
```suggestion
      <Hero />
```

Change this back to the way it was.